packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -67,8 +67,7 @@ const getStyles = (theme: GrafanaTheme2) => {
   return {
     topVerticalAlign: css`
       label: topVerticalAlign;
-      vertical-align: top;
-      margin-top: -${theme.spacing(0.5)};
+      margin-top: -${theme.spacing(0.9)};
```

Comment:
I tried here `-${theme.spacing(1)};`, but `-${theme.spacing(0.9)};` looked better and more aligned.